equinix/resource_fabric_cloud_router.go

```diff
@@ -15,47 +18,250 @@ import (
 
 	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-func resourceCloudRouter() *schema.Resource {
+func FabricCloudRouterPackageSch() map[string]*schema.Schema {
```

Comment:
We can lower case the first letter on this so that it's not exported from the `equinix` package. It becomes a breaking change when the interface changes like that. All Uppercase Methods/Structs/Variables will be exported automatically.